tests/ignite/test_utils.py

```diff
@@ -191,17 +191,15 @@ def func_no_docs():
     # Test on function with docs, @deprecated without reasons
     @deprecated("0.4.2", "0.6.0")
     def func_no_reasons():
-        """Docs are cool
-        """
+        """Docs are cool"""
         return 24
 
     assert func_no_reasons.__doc__ == "**Deprecated function**.\n\n    Docs are cool\n        .. deprecated:: 0.4.2"
```

Comment:
Tests are onw failing here as `\n` is removed in the docstring of `func_no_reasons`. Let's update expected result:
```suggestion
    assert func_no_reasons.__doc__ == "**Deprecated function**.\n\n    Docs are cool        .. deprecated:: 0.4.2"
```

Reply:
@louis-she below expected results are needed to be updated as well, but I'm not 100% sure if just removing "\n" is enough...